core/trino-main/src/main/java/io/trino/sql/planner/planprinter/HashCollisionPlanNodeStats.java

```diff
@@ -39,7 +39,7 @@ public HashCollisionPlanNodeStats(
             long planNodeOutputPositions,
             DataSize planNodeOutputDataSize,
             DataSize planNodeSpilledDataSize,
-            Map<String, OperatorInputStats> operatorInputStats,
+            Map<String, OperatorInputStats> operatorStats,
```

Comment:
Should class be renamed too?

Reply:
Oh I see you rename it in other commit. Should be here.

Reply:
will squash them